src/query/algorithms/entityresolution/BloomFilter.hpp

```diff
@@ -0,0 +1,66 @@
+//
```

Comment:
Need to have proper license header

Reply:
JasmineGraph License header added with year as 2021